zeppelin-web/README.md

```diff
@@ -19,19 +19,27 @@ It is recommended to install node 6.0.0+ since Zeppelin uses 6.9.1+ (see [creati
 
 All build commands are described in [package.json](./package.json)
 
-```sh
 # install required depepdencies and bower packages (only once)
-$ npm install -g yarn
-$ yarn install
+```
+$ npm install -g yarn bower npm-run-all rimraf grunt webpack karma karma-phantomjs-launcher
```

Comment:
I'm don't think 'rimraf grunt webpack karma karma-phantomjs-launcher' are really a required package installed globally for development. Shell we remove them?

Also shell we not use 'npm-run-all' package? 
After this PR, every existing developer will face error, such as

```
$ npm-run-all --parallel dev:server dev:watch 
sh: npm-run-all: command not found
error Command failed with exit code 127.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

and all developer need to read README.md again and install necessary package. which is big impact compare to benefit that we can get from npm-run-all here. 

What do you think?